### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ashwinnair14**
Added badges

### CODE REVIEW

**Review by evanderiel — APPROVED** [review]
Looks good but we should figure out why tests are failing

File: README.md [hunk 1]
```diff
@@ -1,4 +1,16 @@
 [![Python package](https://github.com/mobiusml/aana_sdk/actions/workflows/python-package.yml/badge.svg)](https://github.com/mobiusml/aana_sdk/actions/workflows/python-package.yml)
+![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)
```
Comment (movchan74) [inline README.md:2]: When clicked it doesn't show the license.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+![Build Status](https://img.shields.io/badge/build-passing-brightgreen.svg)
```
Comment (movchan74) [inline README.md:3]: This doesn't make sense. It shows that it's always passing. Maybe let's rename "Python package" badge to "Build Status" and remove that one.

**Review by movchan74 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ashwinnair14) [reply README.md:2]: Done

File: README.md (hunk 2 above)
Comment (ashwinnair14) [reply README.md:3]: Done

Comment (movchan74) [reply README.md]: It still shows up as a "Python package" but I guess it's alright.

**Review by movchan74 — APPROVED** [review]

**Pull request merged**